packages/jest-transform/src/__tests__/__snapshots__/script_transformer.test.ts.snap

```diff
@@ -1,80 +1,98 @@
+          "moduleFileExtensions": Array [
+            "js",
+          ],
+          "moduleLoader": "/test_module_loader_path",
+          "moduleNameMapper": Array [],
+          "modulePathIgnorePatterns": Array [],
+          "modulePaths": Array [],
+          "name": "test",
+          "prettierPath": "prettier",
+          "resetMocks": false,
+          "resetModules": false,
+          "resolver": undefined,
+          "restoreMocks": false,
+          "rootDir": "/",
+          "roots": Array [],
+          "runner": "jest-runner",
+          "setupFiles": Array [],
+          "setupFilesAfterEnv": Array [],
+          "skipFilter": false,
+          "skipNodeResolution": false,
+          "slowTestThreshold": 5,
+          "snapshotResolver": undefined,
+          "snapshotSerializers": Array [],
+          "testEnvironment": "node",
+          "testEnvironmentOptions": Object {},
+          "testLocationInResults": false,
+          "testMatch": Array [],
+          "testPathIgnorePatterns": Array [],
+          "testRegex": Array [
+            "\\\\.test\\\\.js$",
+          ],
+          "testRunner": "jest-jasmine2",
+          "testURL": "http://localhost",
+          "timers": "real",
+          "transform": Array [
+            Array [
+              "\\\\.js$",
+              "test_preprocessor",
+              Object {},
+            ],
+          ],
+          "transformIgnorePatterns": Array [
+            "/node_modules/",
+          ],
+          "unmockedModulePathPatterns": undefined,
+          "watchPathIgnorePatterns": Array [],
+        },
+        "configString": "{\\"automock\\":false,\\"cache\\":true,\\"cacheDirectory\\":\\"/cache/\\",\\"clearMocks\\":false,\\"coveragePathIgnorePatterns\\":[],\\"cwd\\":\\"/test_root_dir/\\",\\"detectLeaks\\":false,\\"detectOpenHandles\\":false,\\"errorOnDeprecated\\":false,\\"extraGlobals\\":[],\\"forceCoverageMatch\\":[],\\"globals\\":{},\\"haste\\":{},\\"injectGlobals\\":true,\\"moduleDirectories\\":[],\\"moduleFileExtensions\\":[\\"js\\"],\\"moduleLoader\\":\\"/test_module_loader_path\\",\\"moduleNameMapper\\":[],\\"modulePathIgnorePatterns\\":[],\\"modulePaths\\":[],\\"name\\":\\"test\\",\\"prettierPath\\":\\"prettier\\",\\"resetMocks\\":false,\\"resetModules\\":false,\\"restoreMocks\\":false,\\"rootDir\\":\\"/\\",\\"roots\\":[],\\"runner\\":\\"jest-runner\\",\\"setupFiles\\":[],\\"setupFilesAfterEnv\\":[],\\"skipFilter\\":false,\\"skipNodeResolution\\":false,\\"slowTestThreshold\\":5,\\"snapshotSerializers\\":[],\\"testEnvironment\\":\\"node\\",\\"testEnvironmentOptions\\":{},\\"testLocationInResults\\":false,\\"testMatch\\":[],\\"testPathIgnorePatterns\\":[],\\"testRegex\\":[\\"\\\\\\\\.test\\\\\\\\.js$\\"],\\"testRunner\\":\\"jest-jasmine2\\",\\"testURL\\":\\"http://localhost\\",\\"timers\\":\\"real\\",\\"transform\\":[[\\"\\\\\\\\.js$\\",\\"test_preprocessor\\",{}]],\\"transformIgnorePatterns\\":[\\"/node_modules/\\"],\\"watchPathIgnorePatterns\\":[]}",
```

Comment:
Looking at this snapshot and thinking if we need to bust the transform cache when any of the configuration options change? Wouldn't it be sufficient to only reset cache when transform output affecting config is changed, like `cache`, `transform`, `cacheDirectory`, `extraGlobals`? Downside would be maintaining such a list

Reply:
>Downside would be maintaining such a list

exactly this. Might be worth it, tho